addons/web/static/src/views/kanban/kanban_controller.js

```diff
@@ -186,6 +195,7 @@ export class KanbanController extends Component {
         const modelConfig = this.props.state?.modelState?.config || {
             resModel,
             activeFields,
+            activeAggregates: this.progressBarAggregateFields.map((field) => field.name),
```

Comment:
should not be in the model's config, imo (only relevant for the kanban controller, to fetch the progressbar data)